bindings/pyroot/src/Cppyy.cxx

```diff
@@ -74,6 +74,8 @@ namespace {
 class ApplicationStarter {
 public:
    ApplicationStarter() {
+      // Initialize explicitly ROOT.
+      (void)gROOT;
```

Comment:
This will need to be added to `clingwrapper.cxx` of cppyy in experimental PyROOT too.
@wlav may want to see this.

Reply:
Done.

Reply:
@wlav after this is merged into ROOT, I can make a PR to cppyy-backend to have this change in cppyy too.